packages/venia-ui/lib/components/RichContent/PageBuilder/utils.js

```diff
@@ -0,0 +1,162 @@
+    let verticalAlignment = null;
+    if (node.style.justifyContent) {
+        verticalAlignment = flexToVerticalAlignment(node.style.justifyContent);
+    }
+
+    return {
+        verticalAlignment
+    };
+}
+
+/**
+ * Convert vertical alignment values to flex values
+ *
+ * @param alignment
+ * @returns {*}
+ */
+export function verticalAlignmentToFlex(alignment) {
+    return alignmentToFlex[alignment];
+}
+
+/**
+ * Convert flex to vertical alignment values
+ *
+ * @param flex
+ * @returns {*}
+ */
+export function flexToVerticalAlignment(flex) {
+    const flexToAlignment = Object.assign(
+        {},
+        ...Object.entries(alignmentToFlex).map(([a, b]) => ({ [b]: a }))
+    );
+    return flexToAlignment[flex];
+}
+
+/**
+ * Retrieve advanced props from content type node
+ *
+ * @param node
+ * @returns {{border: (string|string[]|string), marginRight: (*|string), borderColor: *, paddingBottom: (*|number|string), borderRadius: *, borderWidth: *, paddingRight: (*|number|string), marginBottom: (*|string), paddingTop: (*|string), paddingLeft: (*|string), marginTop: (*|string), marginLeft: (*|string|{get}|number)}}
+ */
+export function getAdvanced(node) {
+    return {
+        ...getPadding(node),
+        ...getMargin(node),
+        ...getBorder(node),
+        ...getTextAlign(node),
+        ...getCssClasses(node)
+    };
+}
+
+/**
+ * Retrieve the padding from a content type node
+ *
+ * @param node
+ * @returns {{paddingBottom: *, paddingRight: *, paddingTop: *, paddingLeft: *}}
+ */
+export function getPadding(node) {
+    return {
+        paddingTop: node.style.paddingTop,
+        paddingRight: node.style.paddingRight,
+        paddingBottom: node.style.paddingBottom,
+        paddingLeft: node.style.paddingLeft
+    };
+}
+
+/**
+ * Retrieve the margin from a content type node
+ *
+ * @param node
+ * @returns {{marginRight: *, marginBottom: *, marginTop: *, marginLeft: *}}
+ */
+export function getMargin(node) {
+    return {
+        marginTop: node.style.marginTop,
+        marginRight: node.style.marginRight,
+        marginBottom: node.style.marginBottom,
+        marginLeft: node.style.marginLeft
+    };
+}
+
+/**
+ * Retrieve the border from a content type node
+ *
+ * @param node
+ * @returns {{border: (string|string), borderColor: *, borderRadius: *, borderWidth: *}}
+ */
+export function getBorder(node) {
+    return {
+        border: node.style.borderStyle,
+        borderColor: node.style.borderColor,
+        borderWidth: node.style.borderWidth,
+        borderRadius: node.style.borderRadius
+    };
+}
+
+/**
+ * Retrieve the text align from a content type node
+ *
+ * @param node
+ * @returns {{textAlign: *}}
+ */
+export function getTextAlign(node) {
+    return {
+        textAlign: node.style.textAlign
+    };
+}
+
+/**
+ * Retrieve the CSS classes from a content type node
+ * @param node
+ * @returns {{cssClasses: any}}
+ */
+export function getCssClasses(node) {
```

Comment:
This can be simplified to:

```js
export function getCssClasses(node) {
   return Array.from(node.classList);
}
```

Reply:
Works because `classList` is a [`DOMTokenList`](https://dom.spec.whatwg.org/#domtokenlist), which is an iterable of strings